services/types.go

```diff
@@ -1,7 +1,16 @@
 package services
 
+type OnboardingSubjectRole string
+
+const (
+	ClientRole OnboardingSubjectRole = "ocs-client"
+	PeerRole   OnboardingSubjectRole = "ocs-peer"
+)
+
 type OnboardingTicket struct {
 	ID                string `json:"id"`
 	ExpirationDate    int64  `json:"expirationDate,string"`
 	StorageQuotaInGiB uint   `json:"storageQuotaInGiB,omitempty"`
+	// OnboardingSubjectRole can be either ocs-client or ocs-peer
```

Comment:
If you are adding a comment please separate using a blank line
```suggestion

	// OnboardingSubjectRole can be either ocs-client or ocs-peer
```